content/module-reference/processing-modules/color-equalizer.md

```diff
@@ -1,7 +1,71 @@
 ---
-title: color equalizer (placeholder)
+title: color equalizer
 id: color-equalizer
 weight: 10
 applicable-version: 4.8
+tags: color color-processing
+working-color-space: UCS
+view: darkroom
+masking: true
+include_toc: true
 ---
 
+Selectively adjust the hue, saturation, and brightness of pixels based on the color.
+
+Color Equalizer is essentially an implementation of the Color Zones module, but works in the scene referred part of darktable's pixelpipe.
+
+This module works in darktable's UCS 22 color space.
+
+he entire hue range is represented as a curve with eight overlapping sections divided by eight fixed nodes. Choose whether you wish to adjust (select) pixels based on their hue, saturation, and brightness. You can then use the equalizer interface nodes, on their respective tabs, to adjust the hue, saturation, and brightness of ranges of pixels selected via this method.
+
+The distances between nodes cannot be changed but all nodes together can be moved by +/- 23 degrees along the color wheel using the node placement slider.
+
+---
+
+**Note**: Because all processing in color  equalizer is based on the hue data of each pixel, it can result in increased chroma noise or even increased luma noise if a lot of brightness changes are applied.
+
+---
+
+# module controls
+
+## hue tab
+
+Click+drag the control points on the curve to change the color of all pixels where the mask includes the given color.
+
+## saturation tab
+
+Click+drag the control points on the curve to change the saturation of all pixels where the mask includes the given color.
+
+## brightness tab
+
+Click+drag the control points on the curve to change the lightness of all pixels where the mask includes the given color.
+
+## node placement
+
+node placement
+: Move the position of the nodes horizontally.
+
```

Comment:
## range selection

To better estimate which nodes should be used for manipulation, you can use the color picker to select the exact point or color range in the photo. 

Activate the color picker and click on one spot in the image. A white vertical line indicates where the pixel is located in the hue range:

![ce_color_picker](https://github.com/user-attachments/assets/953fca88-2aac-4fb0-8125-b813105e6ea4)

If you activate the color picker by holding down the ctrl key, an area can be selected:

![ce_color_picker_2](https://github.com/user-attachments/assets/7770f1f9-d376-4ed6-82b5-93124ad62607)

Reply:
I think it's good to illustrate point and range selection with color pickers, but this might be better in a more generic section, since it will apply to other modules as well.

Reply:
> I think it's good to illustrate point and range selection with color pickers, but this might be better in a more generic section, since it will apply to other modules as well.

I agree. This is an important point and makes sense in terms of general use. Ideally, all modules that offer area selection should do so in the same way.  Parametric masks and color zones have an extra color picker for this.

Global color picker and color equalizer do it via keyboard shortcut.

Reply:
I decided to go minimal and forego the images here too, if you feel strongly we need images, please feel free to add them.